mantidproject/mantid
# Load now produces the history of the loader called

**Description of work.**

The history from Load did not include the correct properties, as it would ignore properties for the specific flavour of the load that was used. This caused loading muon data via "Load" to not produce the correct history. 

**To test:**


Open Muon Analysis
Load some data
Grab the history from any of the workspaces and put it into a script
Clear the ADS
Run the script
All of the workspaces should be reproduced. 


Fixes #32950. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
